src/types/types.ts

```diff
@@ -14,13 +14,6 @@ export interface Intersector {
 	clickableType: string | undefined;
 }
 
-export interface IntersectorConfig {
```

Comment:
I couldn't see this used anywhere so took the opportunity to remove. Let me know if this is ok.

Reply:
Oh, yes. I used that at the beginning. It's not necessary anymore, I forgot to clean it up. Nice job!